src/args.rs

```diff
@@ -21,6 +21,9 @@ pub struct Args {
     #[clap(long, env = "MICROBIN_EDITABLE")]
     pub editable: bool,
 
+    #[clap(long, env = "MICROBIN_NON_REMOVABLE")]
+    pub non_removable: bool,
```

Comment:
Generally it's considered preferable to use positive expressions rather than negated ones. The reason is that it can be confusing when you need to negate it, like you did in the HTML ("if not non_removable"). I'd suggest either renaming this to something like `permanent` or inverting the logic and calling it `removable`.